src/components/MetadataEditor/FormField.vue

```diff
@@ -82,7 +82,6 @@
       :value="value"
       :error="typeof error !== 'string' ? error : null"
       :required="required"
-      :fetchSuggestions="fetchSuggestions"
```

Comment:
Unrelated tidy up - `fetchSuggestions` isn't required by the `<detector-resolving-power-input>` component at all